tools/measurement/apksize/default.gradle

```diff
@@ -30,7 +30,7 @@ android {
     defaultConfig {
         applicationId 'com.google.apksize'
         minSdkVersion project.targetSdkVersion
-	    multiDexEnabled true
+            multiDexEnabled true
```

Comment:
Should this be aligned with the other lines?

Reply:
Done.